Tests/test_PDB.py

```diff
@@ -1231,14 +1257,14 @@ def test_entitity_copy(self):
 
 
 def eprint(*args, **kwargs):
-    '''Helper function that prints to stderr.'''
+    """Helper function that prints to stderr."""
     print(*args, file=sys.stderr, **kwargs)
 
 
 def will_it_float(s):
-    '''
+    """
```

Comment:
With your permission, I've added a summary line and then a modified version of the original text separated with a blank line.

`pydocstyle` hasn't complained on my local machine, let's see what TravisCI says.